src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/Microsoft.VisualStudio.Web.CodeGeneration.Utils.csproj

```diff
@@ -26,13 +26,12 @@
     <PackageReference Include="NuGet.Frameworks" Version="$(NuGetFrameworksVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
-    <PackageReference Include="System.AppContext" Version="$(CoreFxVersion)" />
-    <PackageReference Include="System.Security.Cryptography.Algorithms" Version="$(CoreFxVersion)" />
-    <PackageReference Include="System.Threading.Tasks.Parallel" Version="$(CoreFxVersion)" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="System.Runtime.Loader" Version="$(CoreFxVersion)" />
```

Comment:
We should probably remove this package. It isn't exactly compatible with netstandard2.0. It only happens to install by coincidence, but may cause issues.

FYI AssemblyLoadContext is available on .NET Framework 4.7 and .NET Core, but not ns2.0. This class library may need to target netcoreapp2.0 directly if you need this API.